client-app-example/public/client-app-1/index.html

```diff
@@ -8,13 +8,17 @@ <h1>Component #1</h1>
     <h2>Routing</h2>
     <h3>Current Local Route:</h3>
     <div id="path"></div>
+    <h3>Current Host Route:</h3>
+    <div id="hostPath"></div>
 
     <h3>Change Route</h3>
     <ul>
-      <li><a href="#/app1/route1">This client: /route1</a></li>
-      <li><a href="#/app1/route2">This client: /route2</a></li>
-      <li><a href="#/app2/route1">Other client: /route1</a></li>
-      <li><a href="#/app2/route2">Other client: /route2</a></li>
+      <!-- internal routes URLs get remapped in js by `transformLinks` -->
+      <li><a href="#/route1">This client: /route1</a></li>
+      <li><a href="#/route2">This client: /route2</a></li>
+      <!-- External URLs use full links so they open nicely in new tabs -->
+      <li><a href="localhost:3000/#/app2/route1">Other client: /route1</a></li>
+      <li><a href="localhost:3000/#/app2/route2">Other client: /route2</a></li>
```

Comment:
I updated these links to use `href` fields from the client app's perspective, and then transform as necessary in javascript so folks running the demo can see what the generated links should look like, and when to use code to generate URLs vs just pointing at a new location.